src/Controller/Component/ModulesComponent.php

```diff
@@ -119,6 +119,10 @@ public function getModules()
                 return -$idx;
             })
             ->toList();
+        $plugins = Configure::read('Modules.plugins');
+        if ($plugins) {
+            $modules = array_merge($modules, $plugins);
```

Comment:
Could you please add a test case for this line?

I'm afraid you'll have to add a fourth argument to `ModulesComponentTest::testGetModules()`.